### PR TITLE
Fix formula, add offset angle

### PR DESCRIPTION
**Pull request by jakic12**

### CODE REVIEW

File: src/components/Atmosphere.jsx [hunk 1]
```diff
@@ -9,12 +9,22 @@ class Atmosphere extends React.Component {
...
 
+	componentDidMount() {
+		document.addEventListener('mouseup', this.handleMouseUp);
+		document.addEventListener('mousemove', this.handleRotate);
+	}
+
+	componentWillUnmount() {
+		document.removeEventListener('mouseup', this.handleMouseUp);
+		document.removeEventListener('mousemove', this.handleRotate);
+	}
```
Comment (plojyon) [inline src/components/Atmosphere.jsx]: Why is this better?

File: src/components/Atmosphere.jsx [hunk 2]
```diff
@@ -38,17 +48,21 @@ class Atmosphere extends React.Component {
...
         }
 
-        const bounds = (this.ref ? this.ref.current.getBoundingClientRect() : { left: 0, top: 0, width: 0, height: 0 });
-        const centerX = bounds.left + (bounds.width / 2);
-        const centerY = bounds.top + (bounds.height / 2);
+        this.setState({ angle: this.getMouseAngle(e) + this.state.angleOffset });
+    }
+
+	getMouseAngle = (e) => {
+		const bounds = (this.ref ? this.ref.current.getBoundingClientRect() : { left: 0, top: 0 });
```
Comment (plojyon) [inline src/components/Atmosphere.jsx]: You removed ` width: 0, height: 0 }`

**Review by plojyon — CHANGES_REQUESTED** [review]

File: src/components/Atmosphere.jsx (hunk 2 above)
Comment (jakic12) [reply src/components/Atmosphere.jsx]: Oops

File: src/components/Atmosphere.jsx (hunk 1 above)
Comment (jakic12) [reply src/components/Atmosphere.jsx]: image

Comment (plojyon) [reply src/components/Atmosphere.jsx]: did not read :skull:

**Review by plojyon — APPROVED** [review]
lgtm

**Pull request merged**